packages/avs-bot/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "avs-bot",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
```

Comment:
We are not using MIT license. Please use "GPL-3.0".

Reply:
https://github.com/wireapp/wire-web-packages/pull/3070/commits/304893e51492c9f05400c145efa493c72255371a